runio.js

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env node
+const fs = require('fs');
+const path = require('path');
+const { stopOnFail, chdir, git, copy, exec, replaceInFile, npmRun, npx, writeToFile, runio } = require('runio');
+
+stopOnFail();
+
+module.exports = {
+  async docs() {
+    await Promise.all([
+      this.docsHelpers(),
+      this.docsPlugins(),
+      this.docsExternalHelpers(),
+    ]);
+
+  },
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must not be padded by blank lines. (padded-blocks)](https://app.codacy.com/app/DavertMik/CodeceptJS/pullRequest?prid=4496293)